cmplibrary/src/main/java/com/sourcepoint/cmplibrary/ConsentWebView.java

```diff
@@ -250,7 +316,7 @@ public boolean onCreateWindow(WebView view, boolean dialog, boolean userGesture,
             }
             return false;
         });
-        addJavascriptInterface(new MessageInterface(), "JSReceiver");
```

Comment:
I first thought about renaming `JSReceiver` to `SDK` as well. But I think having a different name to reference the javascript code that our SDK injects in the WebView is useful and I decided to keep `JSReceiver`. What you think?

Reply:
lol I also think so, was just using SDK to try to be concise with the IOs naming as well.